src/main/java/frc/config/Config.java

```diff
@@ -45,30 +66,34 @@ public static double getDouble(String key) {
 	public static void start() {
 		map = new HashMap<>();
 		try {
+			// read the configuration from a file
 			Scanner nameScanner = new Scanner(new File("/home/lvuser/name.192"));
 			String name = nameScanner.nextLine();
 			nameScanner.close();
 			String fileName = name + ".txt";
 			System.out.println("reading from file " + fileName);
 			File f = new File(Filesystem.getDeployDirectory(), fileName);
 			Scanner scanner = new Scanner(f);
+			// for each line of the config file, add a new key/value pair
+			// into the map
```

Comment:
It might be helpful to also explain how information in the config file is organized. For example are values comma-separated, fixed-width columns, or some other delimiter?
This information helps when reading the code below.